### PR TITLE
Add transport debugging option for sockjs debugging

### PR DESCRIPTION
**Pull request by jonyoder**
This option works for old `sockjs-client` versions, which we're currently using. When we update to a current sockjs-client, this option will have no effect.

### CODE REVIEW

File: dist/shiny-server-client.js [hunk 1]
```diff
@@ -1953,9 +1954,14 @@ exports.createFactory = function (options) {
       }
     });
 
-    var conn = new global.SockJS(url, null, {
-      protocols_whitelist: whitelist
-    });
+    var transportDebugging = options.transportDebugging && options.transportDebugging == true;
```
Comment (aronatkins) [inline dist/shiny-server-client.js]: Maybe just blindly pass it down if it is specified; let the sockjs library interpret its meaning.

**Comment by aronatkins** [comment]
🐞

File: dist/shiny-server-client.js (hunk 1 above)
Comment (jonyoder) [reply dist/shiny-server-client.js]: Yeah, that's a bit ugly, but it's an attempt to force a boolean value. I hate to blindly pass down a value that is _intended_ to be bool.

Comment (jcheng5) [reply dist/shiny-server-client.js]: Is this a simple boolean cast of `options.transportDebugging`? Is there a reason you didn't just do `options.transportDebugging == true`, or `!!options.transportDebugging`?

Other than that, LGTM. Feel free to merge when ready.

**Pull request merged**